cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -271,8 +271,8 @@ module connectionCreatorIdentity 'br/public:avm/res/managed-identity/user-assign
   }
 }
 {{- end}}
-{{- if (and .DbPostgres (eq .DbPostgres.AuthType "USER_ASSIGNED_MANAGED_IDENTITY")) }}
 {{- range .Services}}
+{{- if (and .DbPostgres (eq .DbPostgres.AuthType "USER_ASSIGNED_MANAGED_IDENTITY")) }}
```

Comment:
bugfix: when Postgres or MySQL detected, should only create connection for those services that use DB, not all services.